sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatClientBuilder.java

```diff
@@ -178,8 +178,8 @@ public ChatAsyncClient buildAsyncClient() {
             final HttpPipelinePolicy authorizationPolicy;
             if (this.communicationTokenCredential != null) {
                 authorizationPolicy = chain -> {
-                    final Future<AccessToken> tokenFuture = this.communicationTokenCredential.getToken();
-                    final AccessToken token;
+                    final Future<CommunicationAccessToken> tokenFuture = this.communicationTokenCredential.getToken();
```

Comment:
Are we still using Future here instead of CompletableFuture?

Reply:
this is addressed.